pkg/engine/variables/operator/numeric.go

```diff
@@ -37,14 +38,30 @@ func compareByCondition(key float64, value float64, op kyverno.ConditionOperator
 		return key <= value
 	case kyverno.LessThan:
 		return key < value
-	case kyverno.Equals:
-		return key == value
-	case kyverno.Equal:
-		return key == value
-	case kyverno.NotEquals:
-		return key != value
-	case kyverno.NotEqual:
-		return key != value
+	// case kyverno.Equals:
+	// 	return key == value
+	// case kyverno.Equal:
+	// 	return key == value
+	// case kyverno.NotEquals:
+	// 	return key != value
+	// case kyverno.NotEqual:
+	// 	return key != value
```

Comment:
I believe these conditions are no more needed because we have separate files to handle `Equal` and `NotEqual` conditions.
equal.go: https://github.com/kyverno/kyverno/blob/main/pkg/engine/variables/operator/equal.go
notequal.go: https://github.com/kyverno/kyverno/blob/main/pkg/engine/variables/operator/notequal.go

Reply:
Ok, please delete them if they are not required.